modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -36,27 +38,27 @@ spec:
   }'
 ----
 <1> The name for the `NetworkAttachmentDefinition` object.
-<2> Optional: Annotation key-value pair for node selection, where `bridge-interface` must match the name of a bridge configured on some nodes. If you add this annotation to your network attachment definition, your virtual machine instances will only run on the nodes that have the `bridge-interface` bridge connected.
-<3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
-<4> The actual name of the Container Network Interface (CNI) plug-in that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
+<2> Optional: Annotation key-value pair for node selection, where `bridge-interface` must match the name of a bridge configured on some nodes. If you add this annotation to your `NetworkAttachmentDefinition`, your virtual machine instances will only run on the nodes that have the `bridge-interface` bridge connected.
+<3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the `NetworkAttachmentDefinition`.
+<4> The actual name of the Container Network Interface (CNI) plug-in that provides the network for this `NetworkAttachmentDefinition`. Do not change this field unless you want to use a different CNI.
 <5> The name of the Linux bridge configured on the node.
 <6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
-<7> Optional: The VLAN tag.
+<7> Optional: The VLAN tag. No additional VLAN configuration is required on the `NodeNetworkConfigurationPolicy`.
```

Comment:
Maybe move the Note text here stating that this is the better way to configure a VLAN?